contrib/bloop/src/mill/contrib/bloop/BloopImpl.scala

```diff
@@ -206,6 +216,8 @@ class BloopImpl(ev: () => Evaluator, wd: Path) extends ExternalModule { outer =>
               mode = m.releaseMode() match {
                 case ReleaseMode.Debug => BloopConfig.LinkerMode.Debug
                 case ReleaseMode.Release => BloopConfig.LinkerMode.Release
+                case ReleaseMode.ReleaseFast => BloopConfig.LinkerMode.Release
```

Comment:
Got a non-exhaustive pat-match warning, figured I'd fix it. Bloop config doesn't seem to have corresponding enum values though